src/bentoml/__init__.py

```diff
@@ -214,4 +216,5 @@
     "load_config",
     "save_config",
     "set_serialization_strategy",
+    "types"
```

Comment:
We probably don't need to add this, since we're probably deprecating this anyway.

Maybe we should just put it back in TYPE_CHECKING?